Chrome_Extenstion/FrontEndFiles/HTML/newtab.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>New Tab</title>
+    <link rel="stylesheet" href="../CSS/newtab.css">
+</head>
+<body id="main_body" >
+<h1 id="Title">Welcome</h1>
+<div></div>
```

Comment:
Looks like you left out blank divs here. Im assuming you were trying to create a new line. If thats the case you could use <br/> tags for line break. You could also try doing such operations with CSS mostly by manipulating display properties.